### ISSUE TITLE
Low quality icon

### ISSUE THREAD

**Issue by popey456963** [nylas/nylas-mail#3615]
##### Are there any related issues?
Not that I can find

##### What operating system are you using?
Windows

##### What version of Nylas Mail are you using?
2.0.10

--

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
Nope

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Nope

##### Is the issue reproducible with a particular attachment, message, signature, etc?
Unsure, haven't got another Windows computer to test it on at the moment

Basically, for some reason the desktop icon for Nylas Mail is incredibly low quality, despite there being many high quality icon versions distributed (seemingly) throughout the rest of the project.  None of my other icons are low quality, have an example image:

![](https://puu.sh/wSkSe/8626fd5493.png)

**Comment by CHarrisMedia** [comment]
Have the exact same issue, it seems like the wrong icon is being used. If you go to the program files, there is a .ico called _app.ico_ this seems like the icon it should be using. 

1. Right click shortcut icon
2. Go to properties
3. Click "Change program icon"
4. Select the app.ico icon in the program folder.

This isn't effective when the program is running, that requires changes to the exe.

https://i.imgur.com/jMb96t0.png

This screenshot didn't turn out perfect but shows the wrong icon still being used, see bottom right and top left. https://i.imgur.com/iJ7jEPD.png
